source/jormungandr/tests/chaos_disruptions_tests.py

```diff
@@ -133,6 +134,48 @@ def test_disruption_on_stop_area_b(self):
 
 
 @dataset([("main_routing_test", ['--BROKER.rt_topics='+chaos_rt_topic, 'spawn_maintenance_worker'])])
+class TestChaosDisruptionsLineSection(ChaosDisruptionsFixture):
+    """
+    Note: it is done as a new fixture, to spawn a new kraken, in order not the get previous disruptions
+    """
+    def test_disruption_on_line_section(self):
+        """
+        when calling the pt object line:A, at first we have no disruptions,
+
+        then we mock a disruption sent from chaos, and we call again the pt object line:A
+        we then must have one disruption
+        """
+        self.wait_for_rabbitmq_cnx()
+        response = self.query_region('lines/A')
+
+        lines = get_not_null(response, 'lines')
+        assert len(lines) == 1
+        line = lines[0]
+        #at first no disruption
+        assert 'disruptions' not in line
+
+        status = self.query_region('status')
+        last_loaded_data = get_not_null(status['status'], 'last_rt_data_loaded')
+
+        self.send_chaos_disruption("bobette_the_disruption", "A", "line_section", start="stopA", end="stopB")
+
+        #we sleep a bit to let kraken reload the data
+        self.poll_until_reload(last_loaded_data)
+
+        #and we call again, we must have the disruption now
+        response = self.query_region('lines/A')
+        lines = get_not_null(response, 'lines')
+        assert len(lines) == 1
+        line = lines[0]
+
+        disruptions = get_not_null(line, 'disruptions')
+
+        #at first we got only one disruption
+        assert len(disruptions) == 1
+        assert any(d['uri'] == 'bobette_the_disruption' for d in disruptions)
```

Comment:
it would be nice to test the line section disruption structure, either in this test or in the disruption_test